packages/apollo-gateway/src/datasources/LocalGraphQLDataSource.ts

```diff
@@ -9,14 +9,14 @@ import {
 import { enableGraphQLExtensions } from 'graphql-extensions';
 import { GraphQLDataSource } from './types';
 
-export class LocalGraphQLDataSource implements GraphQLDataSource {
+export class LocalGraphQLDataSource<TContext = any> implements GraphQLDataSource<TContext> {
```

Comment:
Context shouldn't ever actually be `any` - it _will_ be treated as an object. In other places, we've been going with:
```suggestion
export class LocalGraphQLDataSource<TContext extends Record<string, any> = Record<string, any>> implements GraphQLDataSource<TContext> {
```